src/portable/portablectl.c

```diff
@@ -769,13 +770,50 @@ static int maybe_stop_enable_restart(sd_bus *bus, sd_bus_message *reply) {
         return 0;
 }
 
-static int maybe_stop_disable(sd_bus *bus, char *image, char *argv[]) {
+static int maybe_clean_units(sd_bus *bus, char **units) {
+        int r;
+
+        assert(bus);
+        assert(units);
```

Comment:
Instead,
```c
if (strv_isempty(units))
        return 0;
```
is better? To gracefully handle the case that the previous method call in `maybe_stop_disable_clean()` provides no unit.

Reply:
will just remove the assert given the foreach will be a no-op in that case